examples/metrics/src/main/java/io/opentelemetry/example/metrics/DoubleCounterExample.java

```diff
@@ -0,0 +1,57 @@
+package io.opentelemetry.example.metrics;
+
+import io.opentelemetry.OpenTelemetry;
+import io.opentelemetry.context.Scope;
+import io.opentelemetry.metrics.DoubleCounter;
+import io.opentelemetry.metrics.LongCounter;
+import io.opentelemetry.metrics.Meter;
+import io.opentelemetry.trace.Span;
+import io.opentelemetry.trace.Span.Kind;
+import io.opentelemetry.trace.Status;
+import io.opentelemetry.trace.Tracer;
+import java.io.File;
+import javax.swing.filechooser.FileSystemView;
+
+/**
+ * Example of using {@link DoubleCounter} and {@link DoubleCounter.BoundDoubleCounter} to count used space by file  with specific extension.
```

Comment:
Actually you don't use the `BoundDoubleCounter` or labels here at all, or am I missing something? 😅 

The file extension would actually make a great metric dimension/label in your example.

```
disk_space_used[file_extension=dll]:   5 MB
disk_space_used[file_extension=pdf]:   1 MB
disk_space_used[file_extension=png]: 710 MB
```
and so on.

What do you think? 🙂

Reply:
Sounds good, when i will be at home i'll try to implement it

Reply:
@arminru when u have a time, pls take a look